trymito.io/pages/python-ai-tools.tsx

```diff
@@ -0,0 +1,223 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from 'next/image';
+import Link from 'next/link';
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import pageStyles from '../styles/Page.module.css';
+import pythonAIToolsStyles from '../styles/PythonAITools.module.css';
+import titleStyles from '../styles/Title.module.css';
+import spreadsheetAutomationStyles from '../styles/SpreadsheetAutomation.module.css';
+import textImageSplitStyles from '../styles/TextImageSplit.module.css';
+
+// Import Icons & Background Grid
+import DownloadCTACard from '../components/CTACards/DownloadCTACard';
+import { classNames } from '../utils/classNames';
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import FAQCard from '../components/FAQCard/FAQCard';
+
+const PythonAITools: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>AI Tools to Write Python Code | Mito</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Mito AI is a notebook-native library with a suite of AI tools for Python and Pandas. Generate, automate, validate, visualize and learn Python with Mito. | Mito" />
+      </Head>
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+          <section className={classNames(titleStyles.title_card, pageStyles.background_card, spreadsheetAutomationStyles.flex_col_mobile_row_desktop)}>
+            <div className={spreadsheetAutomationStyles.title_text_container}>
+              <h1>
+                Write Python and Pandas with AI
```

Comment:
What if we move our unique value prop up? Something like "An AI assistant that helps you write _correct_ code" or something